net-vpn/logmein-hamachi/logmein-hamachi-2.1.0.198.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit linux-info systemd
+
+DESCRIPTION="LogMeIn Hamachi VPN tunneling engine"
+HOMEPAGE="https://www.vpn.net/"
+SRC_URI="x86?	( https://www.vpn.net/installers/${P}-x86.tgz )
+	amd64?	( https://www.vpn.net/installers/${P}-x64.tgz )"
+
+LICENSE="LogMeIn"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE=""
+
+RDEPEND="!net-misc/hamachi"
+
+RESTRICT="mirror"
+
+QA_PREBUILT="/opt/${PN}/bin/hamachid"
+QA_PRESTRIPPED="/opt/${PN}/bin/hamachid"
+QA_WX_LOAD="/opt/${PN}/bin/hamachid"
+
+pkg_setup() {
+	einfo "Checking your kernel configuration for TUN/TAP support."
+	CONFIG_CHECK="~TUN"
+	check_extra_config
+}
+
+src_unpack() {
+	unpack ${A}
+	mv "${P}-$(use x86 && echo x86 || echo x64)" "${S}" || die
```

Comment:
Look into using `usex` here.

Reply:
OK, I will.

I added the irrelevant changes to make repoman happy. Since they aren't important I didn't commit them alone, is it wrong? What would be the preferred way to act?

I didn't rename the old license, they are actually both new. I left the latest with the old name because I think it will probably stay, while the older could be removed when the older package using it won't be available any more on the servers.